tests/unit/dataframer/test_dataframer_smmart.py

```diff
@@ -0,0 +1,232 @@
+        'Observation/21f3411d-89a4-4bcc-9ce7-b76edb1c745f': {'identifier': None, 'resourceType': 'Observation',
+                                                             'id': '21f3411d-89a4-4bcc-9ce7-b76edb1c745f',
+                                                             'status': 'final', 'category': 'Laboratory',
+                                                             'code': 'Genomic structural variant copy number',
+                                                             'Gene': 'TP53', 'Chromosome': 'chr17',
+                                                             'result': 'gain of function (GOF)'},
+        'ResearchStudy/7dacd4d0-3c8e-470b-bf61-103891627d45': {'identifier': 'labA', 'resourceType': 'ResearchStudy',
+                                                               'id': '7dacd4d0-3c8e-470b-bf61-103891627d45',
+                                                               'name': 'LabA', 'status': 'active',
+                                                               'description': 'LabA Clinical Trial Study: FHIR Schema Chorot Integration'},
+        'ResearchSubject/2fc448d6-a23b-4b94-974b-c66110164851': {'identifier': 'subjectX_1234',
+                                                                 'resourceType': 'ResearchSubject',
+                                                                 'id': '2fc448d6-a23b-4b94-974b-c66110164851',
+                                                                 'status': 'active'},
+        'Organization/89c8dc4c-2d9c-48c7-8862-241a49a78f14': {'identifier': 'LabA_ORGANIZATION',
+                                                              'resourceType': 'Organization',
+                                                              'id': '89c8dc4c-2d9c-48c7-8862-241a49a78f14',
+                                                              'type': 'Educational Institute'},
+        'Patient/bc4e1aa6-cb52-40e9-8f20-594d9c84f920': {'identifier': 'patientX_1234', 'resourceType': 'Patient',
+                                                         'id': 'bc4e1aa6-cb52-40e9-8f20-594d9c84f920', 'active': True}}
+
+
+@pytest.fixture()
+def smmart_local_db():
+    """Load a local db with smmart data fixture."""
+    with tempfile.TemporaryDirectory() as temp_dir:
+        # print(f"Temporary directory created at: {temp_dir}")
+        db = LocalFHIRDatabase(db_name=os.path.join(temp_dir, 'local.db'))
+        fixture_path = pathlib.Path('tests/fixtures/fhir-compbio-examples/META')
+        assert fixture_path.exists(), f"Fixture path {fixture_path.absolute()} does not exist."
+        for file in fixture_path.glob('*.ndjson'):
+            # print(f"Loading {file}")
+            for resource in read_ndjson_file(str(file)):
+                db.insert_data_from_dict(resource)
+        yield db
+
+
+@pytest.fixture()
+def document_reference_dataframe():
+    # TODO - is this the expected output?
```

Comment:
Missing...
- `contentType` and `url`. Will defer to @teslajoy on what else in content needs to be surfaced

Reply:
no you are correct. 
should be: 

```
'DocumentReference/9ae7e542-767f-4b03-a854-7ceed17152cb': {'identifier': '9ae7e542-767f-4b03-a854-7ceed17152cb',
                                                                   'resourceType': 'DocumentReference',
                                                                   'id': '9ae7e542-767f-4b03-a854-7ceed17152cb',
                                                                   'status': 'current', 'docStatus': 'final',
                                                                   'date': '2024-08-21T10:53:00+00:00', 
                                                                   'md5': '227f0a5379362d42eaa1814cfc0101b8', 
                                                                   'source_path': 'file:///home/LabA/specimen_1234_labA.fq.gz', 
                                                                    'contentType': 'text/fastq', 
                                                                    'size': 5595609484, 
                                                                     'title': 'specimen_1234_labA.fq.gz', 
                                                                     'creation':  '2024-08-21T10:53:00+00:00'}
```

Reply:
done